glide.yaml

```diff
@@ -9,3 +9,7 @@ import:
 - package: github.com/spf13/cobra
 - package: github.com/xeipuuv/gojsonschema
   version: d5336c75940ef31c9ceeb0ae64cf92944bccb4ee
+- package: github.com/russross/blackfriday
```

Comment:
hmm, just added the "testImports" section, and indeed `glide-vc` doesn't handle this, see https://github.com/sgotti/glide-vc/blob/master/gvc.go#L74.

The resulting `vendor` directory stays the same (it will also include the test dependencies), but I'll submit a PR towards glide-vc to also consider `testImports`.

So I'd say we go with this, once `testImports` is fixed in `glide-vc`, I'll submit a followup-PR.